databricks/koalas/plot/core.py

```diff
@@ -598,69 +589,27 @@ def bar(self, x=None, y=None, **kwds):
 
         Returns
         -------
-        axes : :class:`matplotlib.axes.Axes` or :class:`numpy.ndarray`
-            Return an ndarray when ``subplots=True``.
-            Return an custom object when ``backend!=matplotlib``.
-
+        :class:`plotly.graph_objs.Figure`
+            Return an custom object when ``backend!=plotly``.
+            Return an ndarray when ``subplots=True`` (matplotlib-only).
 
         Examples
         --------
         Basic plot.
 
         For Series:
 
-        .. plot::
-            :context: close-figs
+        .. plotly::
 
             >>> s = ks.Series([1, 3, 2])
-            >>> ax = s.plot.bar()
+            >>> s.plot.bar()  # doctest: +SKIP
 
         For DataFrame:
 
-        .. plot::
-            :context: close-figs
+        .. plotly::
 
             >>> df = ks.DataFrame({'lab': ['A', 'B', 'C'], 'val': [10, 30, 20]})
-            >>> ax = df.plot.bar(x='lab', y='val', rot=0)
-
-        Plot a whole dataframe to a bar plot. Each column is assigned a
-        distinct color, and each row is nested in a group along the
-        horizontal axis.
-
-        .. plot::
-            :context: close-figs
-
-            >>> speed = [0.1, 17.5, 40, 48, 52, 69, 88]
-            >>> lifespan = [2, 8, 70, 1.5, 25, 12, 28]
-            >>> index = ['snail', 'pig', 'elephant',
-            ...          'rabbit', 'giraffe', 'coyote', 'horse']
-            >>> df = ks.DataFrame({'speed': speed,
-            ...                    'lifespan': lifespan}, index=index)
-            >>> ax = df.plot.bar(rot=0)
-
-        Instead of nesting, the figure can be split by column with
-        ``subplots=True``. In this case, a :class:`numpy.ndarray` of
-        :class:`matplotlib.axes.Axes` are returned.
-
-        .. plot::
-            :context: close-figs
-
-            >>> axes = df.plot.bar(rot=0, subplots=True)
-            >>> axes[1].legend(loc=2)  # doctest: +SKIP
-
-        Plot a single column.
-
-        .. plot::
-            :context: close-figs
-
-            >>> ax = df.plot.bar(y='speed', rot=0)
-
-        Plot only selected categories for the DataFrame.
-
-        .. plot::
-            :context: close-figs
-
-            >>> ax = df.plot.bar(x='lifespan', rot=0)
```

Comment:
I removed because `rot` and `subplots` are matplotlib specific arguments. Especially for `subplots` , we should create a container to include other plots via plotly so the way is pretty different.

For `rot=0`, I think probably we can just remove it here .. let me try to recover the examples partially.